bundler/lib/bundler/injector.rb

```diff
@@ -269,5 +270,12 @@ def cross_check_for_errors(gemfile_path, original_deps, removed_deps, initial_ge
     def show_warning(message)
       Bundler.ui.info Bundler.ui.add_color(message, :yellow)
     end
+
+    def convert_autorequire(autorequire)
+      require 'json'
+      autorequire = autorequire.one? ? autorequire.first : autorequire # Unwrap array
+      autorequire = false if autorequire == "false"                    # Convert false to boolean
+      autorequire.to_json                                              # Wrap in quotes if necessary
```

Comment:
I think we don't really need `json` here. Would this be enough?

```ruby
    def convert_autorequire(autorequire)
      autorequire = autorequire.first
      return autorequire if autorequire == "false"
      autorequire.inspect
    end
```

Reply:
It passes the tests, so it'd seem so 👍 

The ethos of this project has been pretty interesting relative to other stuff I've worked on. Things like being quite slim on dependencies and ensuring support for all versions have meant I've had to think a little differently, and that's refreshing!